JSDemos/Demos/Map/ProvidersAndTypes/Angular/app/app.component.html

```diff
@@ -1,11 +1,11 @@
 <dx-map
   #map
-  center="Brooklyn Bridge,New York,NY"
+  center="40.7061, -73.9969"
```

Comment:
y did u change this?

Reply:
It requires to show certain place on the map for testing. If we write text name then place don't appear right.